services/terraform/eks-provision.tf

```diff
@@ -189,6 +211,57 @@ resource "null_resource" "coredns_restart_on_fargate" {
   ]
 }
 
+# ---------------------------------------------------------------------------------------------
+# Fargate logging by fluentbit requires namespace aws-observability and Configmap
+# ---------------------------------------------------------------------------------------------
+
+# Configure Kubernetes namespace aws-observability by adding the aws-observability annotation. This
+# annotation is supported in terraform 0.13 or higher. So kubectl is used to provision the namespace.
+data "template_file" "logging" {
+  template = <<-EOF
+kind: Namespace
```

Comment:
We changed from `<<EOF` to `<<-EOF` so you could indent this template... Can you do that?